managers/backup-manager/BackupService.js

```diff
@@ -132,6 +132,7 @@ class BackupService extends BaseDirectorService {
             resourceId: result.backup_guid,
             status: {
               state: CONST.APISERVER.RESOURCE_STATE.FAILED,
+              response: utils.buildErrorJson(err),
```

Comment:
Will this have state attribute as part of response?

Reply:
Now added state explicitly.